server/data/words/turkish.json

```diff
@@ -0,0 +1,4574 @@
+  },
+  {
+    "word": "fasulye",
+    "translation": "bean"
+  },
+  {
+    "word": "ours",
+    "translation": "bear"
+  },
+  {
+    "word": "battre",
+    "translation": "beat"
+  },
+  {
+    "word": "güzel",
+    "translation": "beautiful"
+  },
+  {
+    "word": "güzellik",
+    "translation": "beauty"
+  },
+  {
+    "word": "çünkü",
+    "translation": "because"
+  },
+  {
+    "word": "olmak",
+    "translation": "become"
+  },
+  {
+    "word": "yatak",
+    "translation": "bed"
+  },
+  {
+    "word": "yatak odası",
+    "translation": "bedroom"
+  },
+  {
+    "word": "bira",
+    "translation": "beer"
+  },
+  {
+    "word": "önce",
+    "translation": "before"
+  },
+  {
+    "word": "başlamak",
+    "translation": "begin"
+  },
+  {
+    "word": "başlangıç",
+    "translation": "beginning"
+  },
+  {
+    "word": "davranış",
+    "translation": "behavior"
+  },
+  {
+    "word": "arkasında",
+    "translation": "behind"
+  },
+  {
+    "word": "varlık",
+    "translation": "being"
+  },
+  {
+    "word": "inanç",
+    "translation": "belief"
+  },
+  {
+    "word": "inanmak",
+    "translation": "believe"
+  },
+  {
+    "word": "çan",
+    "translation": "bell"
+  },
+  {
+    "word": "ait olmak",
+    "translation": "belong"
+  },
+  {
+    "word": "altında",
+    "translation": "below"
+  },
+  {
+    "word": "kemer",
+    "translation": "belt"
+  },
+  {
+    "word": "bank",
+    "translation": "bench"
+  },
+  {
+    "word": "bükmek",
+    "translation": "bend"
+  },
+  {
+    "word": "altında",
+    "translation": "beneath"
+  },
+  {
+    "word": "fayda",
+    "translation": "benefit"
+  },
+  {
+    "word": "dışında",
+    "translation": "beside"
+  },
+  {
+    "word": "dışında",
+    "translation": "besides"
+  },
+  {
+    "word": "en iyi",
+    "translation": "best"
+  },
+  {
+    "word": "bahis",
+    "translation": "bet"
+  },
+  {
+    "word": "daha iyi",
+    "translation": "better"
+  },
+  {
+    "word": "arasında",
+    "translation": "between"
+  },
+  {
+    "word": "üstünde",
+    "translation": "beyond"
+  },
+  {
+    "word": "büyük",
+    "translation": "big"
+  },
+  {
+    "word": "bisiklet",
+    "translation": "bike"
+  },
+  {
+    "word": "fatura",
+    "translation": "bill"
+  },
+  {
+    "word": "milyar",
+    "translation": "billion"
+  },
+  {
+    "word": "bağlamak",
+    "translation": "bind"
+  },
+  {
+    "word": "biyolojik olarak",
+    "translation": "biological"
+  },
+  {
+    "word": "kuş",
+    "translation": "bird"
+  },
+  {
+    "word": "doğum",
+    "translation": "birth"
+  },
+  {
+    "word": "doğum günü",
+    "translation": "birthday"
+  },
+  {
+    "word": "ısırmak",
+    "translation": "bite"
+  },
+  {
+    "word": "siyah",
+    "translation": "black"
+  },
+  {
+    "word": "kılıç",
+    "translation": "blade"
+  },
+  {
+    "word": "suçlamak",
+    "translation": "blame"
+  },
+  {
+    "word": "battaniye",
+    "translation": "blanket"
+  },
+  {
+    "word": kör",
```

Comment:
Changes look good so far -- it seems like you are missing a quotation mark here which is why Github registers the JSON as invalid. You can use [JSONlint](https://jsonlint.com/) to validate if your JSON is correct.

Let me know if this PR is up for review already :)